.git-blame-ignore-revs

```diff
@@ -1,2 +1,3 @@
 # ocamlformat bump
-3c98cd3e0c4e1191983b602a8e38a1c92d259891
\ No newline at end of file
+3c98cd3e0c4e1191983b602a8e38a1c92d259891# Upgrade to OCamlformat 0.26.0
```

Comment:
This comment seems to be on the wrong line

Reply:
Fixed.